style.css

```diff
@@ -0,0 +1,356 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+  font-family: 'Open Sans', sans-serif;
+  font-family: 'Roboto', sans-serif;
+  scroll-behavior: smooth;
+  scroll-padding-top: 0;
+}
+
+input[type=text] {   
+  /* Remove First */
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  /* Style */
+  color: #fff;  
+}
+
+input[type=password]{
+  /* Remove First */
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  /* Style */
+  color: #fff; 
+}
+
+input[type=email]{
+  /* Remove First */
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+
+  /* Style */
+  color: #fff; 
+}
+
+html, body {
+  overflow-x: hidden;
+}
+
+.section{
+  min-height: 100vh;
+  padding: 0 7%;
+  padding-top: 20px;  
+}
+
+/*/ ----------- Main Page ----------/*/
+.full-page{
+  width: 100%;
+  min-height: 100vh;
+}
+
+/*/ navbar /*/
+body{
+  background-color: #080008; 
+}
+
+.header{
+  height: 100vh;
+  width: 100%;
+  padding: 0 8%;
+  position: relative;
+}
+
+nav{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo{
+  width: 40px;
+  margin: 0 30px;
+}
+
+.nav-links{
+  padding: 28px 0;
+  margin: 0 30px;
+}
+
+.nav-links li{
+  display: inline-block;
+  margin: 0 20px;
+}
+
+.nav-links li a{
+  color: black; 
+  padding: 5px 0;
+}
+
+.button{
+  background: #00aa2b;
+  color: #fff;
+  padding: 10px 40px;
+  box-shadow: 6px 6px 39px -4px rgba(0,0,0,0.75);
+  border-radius: 3px;
+  cursor: pointer;
+  transition: 0.4s;
+}
+
+.button:hover{
+  background: #34495e;
+  color: #fff;
+}
+
+/*/ Text over video /*/
+.content{
```

Comment:
The class content is a little hard to understand what & where it does something. Does it change the content for the whole site? Or is it content for a specific part of the site? Maybe you can use the semantic tag "main" if it's regarding content for the whole site?